build.rs

```diff
@@ -602,6 +612,7 @@ fn cc(file: &Path, ext: &str, target: &Target, include_dir: &Path, out_file: &Pa
         && target.os != "redox"
         && target.os != "windows"
         && target.arch != "wasm32"
+        && target.os != "uefi"
```

Comment:
It was a mistake here that we use `"windows"` instead of `WINDOWS`, so I understand why you copied that, but we should just use `UEFI` here.